src/js/App/Header/InsightsAbout.js

```diff
@@ -73,7 +73,8 @@ class InsightsAbout extends Component {
                 { name: 'Compliance', path: 'apps/compliance/app.info.json', version: 'N/A' },
                 { name: 'Cost Management', path: 'apps/cost-management/app.info.json', version: 'N/A' },
                 { name: 'Insights', path: 'apps/insights/app.info.json', version: 'N/A' },
-                { name: 'System Comparison', path: 'apps/drift/app.info.json', version: 'N/A' }
+                { name: 'System Comparison', path: 'apps/drift/app.info.json', version: 'N/A' },
+                { name: 'Migrations', path: 'apps/migration-analytics/app.info.json', version: 'N/A' }
```

Comment:
@kruai should this line instead be
`{ name: 'Migration Analytics', path: 'apps/migration-analytics/app.info.json', version: 'N/A' }` ?

Reply:
Yes, it probably should!